submit/submit

```diff
@@ -70,6 +70,15 @@ def error(msg: str) -> NoReturn:
     exit(-1)
 
 
+def print_if_json(text: str):
+    '''Print the text if it is valid JSON, and ignore otherwise'''
+    try:
+        data = json.loads(text)
+        print(json.dumps(data, indent=2))
+    except json.decoder.JSONDecodeError as e:
+        pass
```

Comment:
```suggestion
def print_if_json(text: str):
    '''Print the text if it is valid JSON, and ignore otherwise'''
    try:
        data = json.loads(text)
        print(json.dumps(data, indent=2))
    except json.decoder.JSONDecodeError:
        pass
```

I think we can even merge the 2 lines in the `try` but I leave that up to your interpretation.

Reply:
I'll leave it as two lines as the merged version looks a little cluttered IMO.